lib/mayaUsd/ufe/UsdShaderAttributeHolder.cpp

```diff
@@ -89,13 +91,34 @@ std::string UsdShaderAttributeHolder::defaultValue() const
 
 std::string UsdShaderAttributeHolder::nativeType() const { return _sdrProp->GetType(); }
 
+//! A List of the properties that are known to lack the default value.
+//! Insert the name of the property to avoid logging the warning.
+static const std::unordered_set<std::string> s_ListPropertiesWithoutDefaultValue {
```

Comment:
A list of properties that we have already noticed to lack the default value, to prevent them to produce warnings. 

It will be maintained as need. Is it the correct place to put it? (just above the method that make use of it). @JGamache-autodesk

Reply:
Please initialize this set as empty. It will be used to track the shaders that previously printed a warning. See next comment.